test/unit-tests/ounit_unicode_tests.ml

```diff
@@ -167,15 +167,6 @@ let suites =
                ] );
          ( __LOC__ >:: fun _ ->
            {|$x)|} ==* [ (0, 0, 0, 2, var, "x"); (0, 2, 0, 3, String, ")") ] );
-         ( __LOC__ >:: fun _ ->
```

Comment:
I wonder if these tests can be left, but updating them to the new behavior?

Reply:
The new behavior is different and it's covered elsewhere, in the blackbox-tests. I think these are actually unnecessary.